detector/wordpress.go

```diff
@@ -35,20 +36,38 @@ type WpCveInfos struct {
 
 // WpCveInfo is for wpscan json
 type WpCveInfo struct {
-	ID         string     `json:"id"`
-	Title      string     `json:"title"`
-	CreatedAt  time.Time  `json:"created_at"`
-	UpdatedAt  time.Time  `json:"updated_at"`
-	VulnType   string     `json:"vuln_type"`
-	References References `json:"references"`
-	FixedIn    string     `json:"fixed_in"`
+	ID            string     `json:"id"`
+	Title         string     `json:"title"`
+	CreatedAt     time.Time  `json:"created_at"`
+	UpdatedAt     time.Time  `json:"updated_at"`
+	PublishedDate time.Time  `json:"published_date"`
+	Description   *string    `json:"description"` // Enterprise only
+	Poc           *string    `json:"poc"`         // Enterprise only
+	VulnType      string     `json:"vuln_type"`
+	References    References `json:"references"`
+	Cvss          *Cvss      `json:"cvss"` // Enterprise only
+	Verified      bool       `json:"verified"`
+	FixedIn       *string    `json:"fixed_in"`
+	IntroducedIn  *string    `json:"introduced_in"`
+	Closed        *Closed    `json:"closed"`
 }
 
 // References is for wpscan json
 type References struct {
 	URL     []string `json:"url"`
 	Cve     []string `json:"cve"`
-	Secunia []string `json:"secunia"`
+	YouTube []string `json:"youtube,omitempty"`
+}
```

Comment:
There also seems to be an entry called `exploitdb`.
```suggestion
	YouTube []string `json:"youtube,omitempty"`
	ExploitDB []string `json:"exploitdb,omitempty"`
}
```

```json
      {
        "id": "475404ce-2a1a-4d15-bf02-df0ea2afdaea",
        "title": "WP Google Maps 7.11.00-7.11.17 - Unauthenticated SQL Injection",
        "created_at": "2019-04-02T12:24:45.000Z",
        "updated_at": "2020-11-19T06:03:54.000Z",
        "published_date": "2019-04-02T00:00:00.000Z",
        "vuln_type": "SQLI",
        "references": {
          "url": [
            "https://plugins.trac.wordpress.org/changeset/2061434/wp-google-maps/trunk/includes/class.rest-api.php",
            "https://packetstormsecurity.com/files/159640/"
          ],
          "cve": [
            "2019-10692"
          ],
          "exploitdb": [
            "48918"
          ]
        },
        "verified": false,
        "poc": null,
        "description": null,
        "cvss": null,
        "fixed_in": "7.11.18",
        "introduced_in": null
      },
```